core/app/app.php

```diff
@@ -128,6 +128,13 @@ private function enqueue_assets() {
 		Plugin::$instance->init_common();
 		Plugin::$instance->common->register_scripts();
 
+		wp_register_style(
+			'select2',
+			ELEMENTOR_ASSETS_URL . 'lib/e-select2/css/e-select2.css',
```

Comment:
Add a minified version via `$suffix = ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG || defined( 'ELEMENTOR_TESTS' ) && ELEMENTOR_TESTS ) ? '' : '.min';`

Reply:
You can use `$this->get_js_assets_url()` for this purpose